src/LibraryManager/Microsoft.Web.LibraryManager.csproj

```diff
@@ -7,9 +7,9 @@
     <PackageId>Microsoft.Web.LibraryManager</PackageId>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Minimatch" />
+    <PackageReference Include="Minimatch" NoWarn="NU1603" />
```

Comment:
Just curious, what was this for?

Reply:
This assembly targets netstandard20, but the package has a dependency on some netstandard 1.x prerelease version.  It makes the restore graph super messy and gives a ton of these warnings.  I also ended up suppressing it globally above, with a comment to the issue in that repo (which is vague on details, but I don't know if this will ever get fixed...)